code/src/components/Form.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+
+import { UserName } from './Name'
+import { AgeRadioOption } from './Age'
+import { SelectPlanet } from './SelectPlanet'
+import { BringItem } from './BringItem'
+import { Summary } from './Summery'
+
+
+
+
+export const Form = () => {
+  const [ageGroupSpan, setAgeGroup] = useState('')
+  const [name, setName] = useState('')
+  const [item, setInputItem] = useState('')
+  const [planet, setPlanet] = useState('')
+  const [summary, setSummary] = useState(false)
```

Comment:
One suggestion is to keep these useState in the order they are mounted in the code below. If you check out the states of them all in the React tools in the browser, they are in a different order from how they are displayed on the page. 🙂